app/assets/javascripts/initializers/initializeAllFollowButts.js

```diff
@@ -107,26 +83,29 @@ function handleOptimisticButtRender(butt) {
   } else {
     // Handles actual following of tags/users
     try {
-      //lets try grab the event buttons info data attribute user id
+      // lets try grab the event buttons info data attribute user id
       var evFabUserId = JSON.parse(butt.dataset.info).id;
       var requestVerb = butt.dataset.verb;
-      //now for all follow action buttons
-      document.querySelectorAll('.follow-action-button').forEach(function(fab) {
-        try {
-          //lets check they have info data attributes
-          if (fab.dataset.info) {
-            //and attempt to parse those, to grab that buttons info user id
-            var fabUserId = JSON.parse(fab.dataset.info).id;
-            //now does that user id match our event buttons user id?
-            if (fabUserId && fabUserId === evFabUserId) {
-              //yes - time to assign the same state!
-              assignState(fab, requestVerb);
+      // now for all follow action buttons
+      document
+        .querySelectorAll('.follow-action-button')
+        .forEach(function syncState(fab) {
+          try {
+            // lets check they have info data attributes
+            if (fab.dataset.info) {
+              // and attempt to parse those, to grab that buttons info user id
+              var fabUserId = JSON.parse(fab.dataset.info).id;
+              // now does that user id match our event buttons user id?
+              if (fabUserId && fabUserId === evFabUserId) {
+                // yes - time to assign the same state!
+                assignState(fab, requestVerb);
+              }
             }
+          } catch (err) {
+            return err;
           }
-        } catch (err) {
-          return;
-        }
-      });
+          return null;
```

Comment:
why `return null`? apparently no one is checking the return value of this function